docs/DEVGUIDE.md

```diff
@@ -249,9 +249,17 @@ Because we haven't created any resources in the service-catalog API server yet,
 
     $ kubectl get brokers,serviceclasses,instances,bindings
 
+### Installing a UPS broker
+Service Catalog requires brokers to operate and there is a User Provided
+Service broker (UPS from now on), which allows consumption of existing
+services through the Service Catalog model. It runs as any other broker,
```

Comment:
The verbiage about 'runs as any other broker'.  How about this instead:

_"Just like any other broker, the UPS broker needs to be running somewhere before it can be added to the catalog.  We need to deploy it first by..."_